doc/1.4/language.md

```diff
@@ -3897,6 +3897,12 @@ template itself contain a method that descends into subobjects; the
 implementation of `hard_reset` in `utility.dml`
 demonstrates how this can be done.
 
+The order in which objects are given by a specific `each`-`in` expression is not
+defined, save that it is deterministic. That is, for a particular choice of
```

Comment:
I think "save" here is confusing, but that might just be me

Reply:
Perhaps this is slightly grammatically incorrect and I should be saying "save for that" instead of simply "save that" (I know I've definitely seen both used, but that doesn't mean both are right). Regardless, yeah, "save" can be used to mean "with the only exception of" or "aside from only". I suspect the origin of that usage comes from the idea of "it would be this thing, if it weren't saved by this one exception".  It's one of the less common uses of the word.

Reply:
or "except it is deterministic"

Reply:
ah whatever. Went for "except for that it is deterministic."